lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -1271,17 +1289,12 @@ void _AddColorManagementFragments(HdMaterialNetwork& net)
             if (fileIt == node.parameters.end() || !fileIt->second.IsHolding<SdfAssetPath>()) {
                 continue;
             }
-            auto const&        filenameVal = fileIt->second.Get<SdfAssetPath>();
-            const std::string& resolvedPath = filenameVal.GetResolvedPath();
+            auto const& filenameVal = fileIt->second.Get<SdfAssetPath>();
+            auto const& resolvedPath = filenameVal.GetResolvedPath();
             if (resolvedPath.empty()) {
                 continue;
             }
-            const std::string& assetPath = filenameVal.GetAssetPath();
-            MString            colorRuleCmd;
-            colorRuleCmd.format(
-                "colorManagementFileRules -evaluate \"^1s\";",
-                (!resolvedPath.empty() ? resolvedPath : assetPath).c_str());
```

Comment:
resolvedPath can not be empty here (as per check on line 1294), so removed unnecessary code checking for assetPath.